src/LoggerPlugin.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Http\Client\Plugin;
+
+use Http\Client\Exception;
+use Http\Client\Plugin\Normalizer\Normalizer;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Log request, response and exception for a HTTP Client
+ *
+ * @author Joel Wurtz <joel.wurtz@gmail.com>
+ */
+class LoggerPlugin implements Plugin
+{
+    /**
+     * @var LoggerInterface Logger to log request / response / exception for a http call
+     */
+    private $logger;
+
+    /**
+     * @var Normalizer Normalize request and response to string or array
+     */
+    private $normalizer;
+
+    public function __construct(LoggerInterface $logger, Normalizer $normalizer)
```

Comment:
Please add docblock for the constructor with typehints (description not required).